.github/workflows/import-events.yml

```
@@ -0,0 +1,56 @@
+name: Import MTBO Events
+
+on:
+  repository_dispatch:
+    types: [scraper-update]
+  workflow_dispatch:
+
+concurrency:
+  group: "import-events-${{ github.ref || github.event.workflow_dispatch?.actor || github.event.client_payload?.id }}"
```

actionlint (1.7.12)

[error] got unexpected character '?' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)